reputation_utils.py

```
@@ -0,0 +1,30 @@
+    return '.'.join(reversed(ip.split('.')))
+
+
+def lookup_dnsbl(ip: str) -> bool:
+    """Return True if IP is listed in any DNSBL_HOSTS."""
+    reversed_ip = _reverse_ip(ip)
+    for bl in DNSBL_HOSTS:
+        query = f'{reversed_ip}.{bl}'
+        try:
+            resolver.resolve(query, 'A')
+            # If resolution succeeds, IP is listed
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)